webpack/assets/javascripts/react_app/components/componentRegistry.js

```diff
@@ -148,6 +149,7 @@ const coreComponets = [
   { name: 'PersonalAccessTokens', type: PersonalAccessTokens },
   { name: 'ClipboardCopy', type: ClipboardCopy },
   { name: 'LabelIcon', type: LabelIcon },
+  { name: 'HostStatuses', type: HostStatuses },
```

Comment:
I don't see us mounting it anywhere from Rails, I believe that means we don't need to expose it in componentRegistry.
Or am I missing something?

Reply:
you're right, fixed